libs/platform/src/lib/components/table/table.component.html

```diff
@@ -72,6 +72,7 @@
                 [id]="id+'__'+column.name"
                 [attr.aria-colindex]="colIdx"
                 [class.fd-table__cell--fixed]="_freezableColumns?.includes(column.name)"
+                [class.fd-table__cell--fixed-last]="_isLastFreezableColumn(colIdx)"
```

Comment:
can we simply do the check as this: `column.name === freezeColumnsTo`?

Reply:
You're right! Thank you :)